src/Authenticator/JwtAuthenticator.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         4.0.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Authentication\Authenticator;
+
+use Authentication\Identifier\IdentifierCollection;
+use Authentication\Result;
+use Cake\ORM\Entity;
```

Comment:
> The ORM is already in the required section of composer.

No it's not. Check the discussion in #46.

>  We need at least EntityInterface because this has to be returned by the Result object when getIdentity() is called.

Why is the identity implementing `EntityInterface` a must?  Can't we just have it implement `ArrayAccess` ?  Have Cake's ORM as a hard dependency will dissuade non cake users from using the lib. I would prefer if this plugin was usable outside cakephp too.